config.js

```diff
@@ -50,7 +50,8 @@ var config = {
 
     // Websocket URL (XMPP)
     websocket: 'wss://jitsi-meet.example.com/' + subdir + 'xmpp-websocket',
-    websocketKeepAliveUrl: 'https://jitsi-meet.example.com/' + subdir + '_unlock',
+
+    // websocketKeepAliveUrl: 'https://jitsi-meet.example.com/' + subdir + '_unlock',
```

Comment:
Is this change intentional?

Reply:
Nvm I saw the bug now. All good.

Reply:
Yeah, this is needed only in multishard deployments with haproxy.